Paralayout/ViewDistributionBuilder.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Copyright © 2024 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+
+#if swift(>=5.4)
+@resultBuilder
+public struct ViewDistributionBuilder {
+
+    // Build expressions, which are turned into partial results.
+
+    public static func buildExpression(_ component: ViewDistributionSpecifying) -> [ViewDistributionSpecifying] {
+        return [component]
+    }
+    public static func buildExpression(_ component: CGFloat) -> [ViewDistributionSpecifying] {
+        return [ViewDistributionItem.fixed(component)]
+    }
+    public static func buildExpression(_ component: Double) -> [ViewDistributionSpecifying] {
+        return [ViewDistributionItem.fixed(component)]
+    }
+    public static func buildExpression(_ component: Int) -> [ViewDistributionSpecifying] {
+        return [ViewDistributionItem.fixed(CGFloat(component))]
+    }
```

Comment:
These three methods enable treating raw numbers as fixed positions in the build expression. So instead of `8.fixed` you could just write `8`.

Happy to delete these for explicit parity with existing APIs. Also happy to add this functionality to existing APIs if folk would like.

Reply:
🤔 I agree if we support unspecified numbers they should be fixed, but I _think_ it would be better to keep it explicit. I could see it being very confusing to open a file and see

```swift
applyVerticalSubviewDistribution {
    oneView
    1
    twoView
}
```

Not having this context, I'd probably assume that's a flexible spacer. For now I think probably best to keep it explicit.